config/config.go

```diff
@@ -474,13 +492,21 @@ func (s *DestinationBlockchain) Validate() error {
 	if _, err := url.ParseRequestURI(s.RPCEndpoint); err != nil {
 		return fmt.Errorf("invalid relayer broadcast URL: %w", err)
 	}
+	if s.KMSKeyID != "" {
+		if s.AWSRegion == "" {
+			return errors.New("KMS key ID provided without an AWS region")
+		}
+		if s.AccountPrivateKey != "" {
+			return errors.New("only one of account private key or KMS key ID can be provided")
+		}
+	} else {
+		if len(s.AccountPrivateKey) != relayerPrivateKeyBytes*2 {
+			return errors.New("invalid account private key hex string")
+		}
 
-	if len(s.AccountPrivateKey) != relayerPrivateKeyBytes*2 {
-		return errors.New("invalid account private key hex string")
-	}
-
-	if _, err := hex.DecodeString(s.AccountPrivateKey); err != nil {
-		return fmt.Errorf("invalid account private key hex string: %w", err)
+		if _, err := hex.DecodeString(s.AccountPrivateKey); err != nil {
+			return fmt.Errorf("invalid account private key hex string: %w", err)
+		}
```

Comment:
I think the behavior we're going for here is:
- if `AccountPrivateKey` is set, it must be valid, `KMSKeyID` cannot be set, and `AWSRegion` cannot be set.
- otherwise, both `KMSKeyID` and `AWSRegion` must be set and valid.

For validating the account private key, I recommend we use `crypto.HexToECDSA` to provide more robust validation.